### PR TITLE
serialize: add skip_serializing_if attribute

### PR DESCRIPTION
**Pull request by evaporei**
Fixes: https://github.com/otaviopace/edn-derive/issues/24

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -7,7 +7,7 @@ mod enums;
 mod serialize;
 mod structs;
 
-#[proc_macro_derive(Serialize)]
+#[proc_macro_derive(Serialize, attributes(edn))]
```
Comment (naomijub) [inline src/lib.rs:10]: I was testing and for performance reasons I would think of creating another derive macro like `SerializeWith` or `SerializeAttr` that contains the attribute macros code. It becomes way slower with those nested `fors`.

Comment (otaviopace) [reply src/lib.rs:10]: Really? But performance at compile time right?

Comment (naomijub) [reply src/lib.rs:10]: it took 12% more out of performance

**Comment by naomijub** [comment]
Maybe we close this?

**Comment by evaporei** [comment]
Ok, I'll close for now and if someone wants to implement it, they can comeback and make a better implementation than this one.

**Pull request closed without merging**